cranelift/codegen/meta/src/isa/spirv/mod.rs

```diff
@@ -0,0 +1,61 @@
+use crate::cdsl::cpu_modes::CpuMode;
+use crate::cdsl::instructions::{InstructionPredicateMap, InstructionGroupBuilder};
+use crate::cdsl::isa::TargetIsa;
+use crate::cdsl::regs::{IsaRegs, IsaRegsBuilder, RegBankBuilder, RegClassBuilder};
+use crate::cdsl::settings::{PredicateNode, SettingGroup, SettingGroupBuilder};
+use crate::cdsl::recipes::{EncodingRecipeBuilder, EncodingRecipeNumber, Recipes, Stack};
+
+use crate::shared::Definitions as SharedDefinitions;
+
+
+pub(crate) fn define_recipes(shared_defs: &SharedDefinitions, regs: &IsaRegs) -> Recipes {
+    let formats = &shared_defs.formats;
+
+    // Register classes shorthands.
+    let gpr = regs.class_by_name("GPR");
+
+    Recipes::new()
+}
+
+fn define_registers() -> IsaRegs {
+    let mut regs = IsaRegsBuilder::new();
+
+    let builder = RegBankBuilder::new("VirtualRegs", "")
+        .units(255) // jb-todo: spirv's registers are virtual...
+        .track_pressure(true);
+    let int_regs = regs.add_bank(builder);
+
+    let builder = RegClassBuilder::new_toplevel("GPR", int_regs);
+    regs.add_class(builder);
+
+    regs.build()
```

Comment:
I think this is can be return an empty `IsaRegs`.

Reply:
Done.